source/GooglePlayServicesNuGetReadMe.cshtml

```diff
@@ -0,0 +1,166 @@
+@using System
+@using System.IO
+@using System.Linq
+@using System.Xml.Linq
+
+  @{
+      string[] artifact_version_parts = Model.NuGetVersion
+                                              .Split(new string[] { "-" }, StringSplitOptions.None);
+      string artifact_version = null;
+
+      string artifact_version_release = artifact_version_parts[0];
+      string[] artifact_version_release_parts = artifact_version_release.Split(new string[] { "." }, StringSplitOptions.None);
+      artifact_version = string.Join(".", artifact_version_release_parts, 0, 3);
+
+      if (artifact_version_parts.Length == 1)
+      {
+          // release
+      }
+      if (artifact_version_parts.Length == 2)
+      {
+          artifact_version += "-" + artifact_version_parts[1];
+      }
+  }
+
+## About
+
+<!-- 
+    A description of the package and where one can find more documentation 
+-->
+
+Google.Play.Services, Firebase and ML Kit (GPS-FB-MLKit) Bindings NuGet Package @(Model.NuGetPackageId) 
+
+.NET for Android (formerly Xamarin.Android) GPS-FB-MLKit bindings NuGet Package @(Model.NuGetPackageId) 
+for Maven artifact:
+
+artifact=@(Model.MavenGroupId):@(Model.Name) 
+
+artifact_versioned=@(Model.MavenGroupId):@(Model.Name):@(artifact_version)
+
+nuget=@(Model.Name).@(Model.NuGetVersion)
+
+For more info please visit: 
+
+https://github.com/xamarin/GooglePlayServicesComponents/
+
+
+## How to Use
+
+<!-- 
+    A compelling example on how to use this package with code 
+-->
+
+See samples:
+
+https://github.com/xamarin/GooglePlayServicesComponents/tree/main/samples
```

Comment:
Do we feel our samples are up-to-date and complete enough that we should be recommending them? Or should we just leave this section out?

Reply:
Well... No they are at least outdated... Samples need tons of love.

Reply:
removed